src/GovUk.Education.ExploreEducationStatistics.Public.Data.Scripts/GovUk.Education.ExploreEducationStatistics.Public.Data.Scripts.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <RootNamespace>GovUk.Education.ExploreEducationStatistics.Public.Data.Scripts</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="CliFx" Version="2.3.5" />
+      <PackageReference Include="Dapper" Version="2.1.24" />
```

Comment:
This all looks really good! My only question is why Dapper and not EF? :P

Reply:
This is because Dapper integrates with DuckDB.NET, whereas EF doesn't. Even if there was an integration, it wouldn't necessarily be useful as we're doing a lot of raw commands that are specific to DuckDB.

We'll most likely be using Dapper in the main query functionality as well. The alternative would be to drop down to raw ADO.NET (the low-level abstraction layer), but this has rubbish developer ergonomics so Dapper is preferable.